docs/implementation/character-intelligence-hunter-focused.md

```
@@ -0,0 +1,143 @@
+# Character Intelligence: Hunter's Perspective
+
+## Overview
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)